syntax = "proto3";

package bullseyeindexservice;

service IndexCalc {
    rpc sendRequest (IndexRequest) returns (IndexReply) {}
}

message IndexRequest {
    string index_id = 1;
}

message IndexReply {
    double index_value = 1;
}